src/EmailServiceClient.php

```diff
@@ -207,9 +207,13 @@ protected function reportUnexpectedResponse($response, $uniqueErrorCode)
     private function assertTrustedIp()
     {
         $baseHost = parse_url($this->serviceUri, PHP_URL_HOST);
-        $serviceIp = gethostbyname(
-            $baseHost
-        );
+        $serviceIp = gethostbyname($baseHost);
+        if ($serviceIp == $baseHost) {
```

Comment:
Sorry for not paying attention to this the first time.  I'm confused why it would be exception if they are equal.

Maybe add a comment.

I'll approve it with the assumption that `==` is correct here.

Reply:
It's just the strange way that this function reports an error.